### DISCUSSION TITLE
Is this github organization associated with Google?

### DISCUSSION THREAD

**Discussion by androiddrew** [googleapis/googleapis#865] [Q&A] [answered]
I am honestly a little confused with the use of Google branding in this organization...is this an official google project? It doesn't state in the README.md and has me terribly confused. I see bot activity though that does state this.

I ask because recently I found https://pypi.org/project/google/ ...which doesn't seem to be an official google python client, and I want to pip the right package for working with google protobufs types.

**Comment by parthea** [comment, marked as answer]
The Github organization `googleapis` belongs to Google and is maintained by Googlers (Google Employees). 

I'm not familiar with the PyPI package https://pypi.org/project/google/ and the maintainer listed for that package does not appear to be a Googler.

https://pypi.org/project/googleapis-common-protos/ may be the package you're looking for as it has python classes generated from common protos in this repository, such as `google.api`, `google.rpc` and `google.type`. Please feel free to file an issue in https://github.com/googleapis/python-api-common-protos/issues if you have specific questions about the `googleapis-common-protos` package.
